grails-app/assets/javascripts/csiBenchmark/csiBenchmarkChart.js

```diff
@@ -104,7 +106,10 @@ OpenSpeedMonitor.ChartModules.CsiBenchmarkChart = (function (chartIdentifier) {
             .attr("class", "d3chart-barLabel")
             .style("text-anchor", "middle")
             .attr("x", barWidth / 2)
-            .attr("y", "1.2em");
+            .attr("y", "1.2em")
+            .attr("transform", function (d) {
```

Comment:
The above applies here as well. You already have the calculation in the update part so this can be deleted and only be used there.